pkg/comp-functions/functions/common/instance_namespace.go

```diff
@@ -139,3 +153,70 @@ func createNamespacePermissions(ctx context.Context, instance string, instanceNs
 	}
 	return svc.SetDesiredKubeObjectWithName(roleBinding, instance+"-service-rolebinding", "namespace-permissions")
 }
+
+// DisableBilling deploys a special config map to the appcat control namespace.
```

Comment:
Wouldn't it be easier to understand if we would have only one CM with a list of disabled namespaces? I feel like having a CM per each disabled instance is a bit of an overkill

Reply:
There are various things that would make it a lot harder. It would be somewhat like your URO issue. The main problem would also be to clean up the CM again, if an instance gets deleted. As soon as a claim/composite has a `deletionTimestamp` Crossplane will not trigger the compositions anymore. So we would need some external controller again to actually make sure that the CM is properly synced up with the state of the all the instances.

Also, we could re-use the CM in the future to support other user hidden settings, if they come up.